ask-sdk-model-runtime/ask_sdk_model_runtime/api_client.py

```diff
@@ -0,0 +1,151 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights
+# Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS
+# OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the
+# License.
+#
+import typing
+import requests
+import six
+import json
+
+from urllib3.util import parse_url
+
+from .api_client_response import ApiClientResponse
+
+from .exceptions import ApiClientException
+
+if typing.TYPE_CHECKING:
+    from typing import Callable, Dict, List, Tuple
+    from ask_sdk_model_runtime import ApiClientRequest
+
+
+class ApiClient(object):
```

Comment:
Can we rename this back to `DefaultApiClient` and also have this subclassed from `ApiClient`, so that we can keep it backwards compatible.

Reply:
Sure, will add ApiClient abstract definition and implement `DefaultApiClient` and also do the same with `Serializer`.